### ISSUE TITLE
doc: Describe how to configure loglevels via configuration

### ISSUE THREAD

**Issue by fgheysels** [arcus-azure/arcus.observability#421]
We should have some documentation which informs people how we can configure the loglevels at runtime / via the configuration.

**Comment by stijnmoreels** [comment]
Isn't that Serilog's resposibility? Don't think that we every have extensions for configuring such log levels? We chould, however, link to Serilog's pages, though. 
https://github.com/serilog/serilog/wiki/Configuration-Basics#minimum-level and the `LoggingLevelSwitch`.

**Comment by fgheysels** [comment]
Yes, that is true.
However, I think we can already make an improvement by making sure that the setup of serilog in the templates that we provide, is done by reading logging settings from the configuration.

**Comment by stijnmoreels** [comment]
Yes, was also experimenting with that, but we have to wait until Serilog is fully .NET 6 supported, as mentioned in this issue: https://github.com/arcus-azure/arcus.templates/issues/480#issuecomment-1200705576
